port/omrportpriv.h

```diff
@@ -518,6 +518,8 @@ omrsysinfo_get_open_file_count(struct OMRPortLibrary *portLibrary, uint64_t *cou
 extern J9_CFUNC intptr_t
 omrsysinfo_get_os_description(struct OMRPortLibrary *portLibrary, struct OMROSDesc *desc);
 extern J9_CFUNC BOOLEAN
+omrsysinfo_get_memory_allocation_hint_supported(struct OMRPortLibrary *portLibrary);
+extern J9_CFUNC BOOLEAN
 omrsysinfo_os_has_feature(struct OMRPortLibrary *portLibrary, struct OMROSDesc *desc, uint32_t feature);
```

Comment:
Shouldn't we be using this API instead of introducing a new `omrsysinfo_get_memory_allocation_hint_supported` that is only ever useful on z/OS?

Reply:
I guess you meant to say using `retrieveZOSIARV64InOriginSupported` directly? 

`retrieveZOSIARV64InOriginSupported` uses port library structures (`J9RCE` and `J9CVT`) to perform an OS query. I think this kind of logic should reside in the port library and be exported via `omrsysinfo_get_memory_allocation_hint_supported` to the GC component. Can/Should we invoke `retrieveZOSIARV64InOriginSupported` directly from the GC?

Reply:
Removed `omrsysinfo_get_memory_allocation_hint_supported ` in e05099798042f0f58bd485cf6623957d41e31387